firestore-stripe-payments/functions/src/index.ts

```diff
@@ -282,9 +283,19 @@ exports.createCheckoutSession = functions.firestore
           });
           setupIntentClientSecret = setupIntent.client_secret;
         } else {
-          throw new Error(
-            `Mode '${mode} is not supported for 'client:mobile'!`
-          );
+          const subscription = await stripe.subscriptions.create({
+            customer,
+            items: [{ plan: price }],
```

Comment:
```suggestion
            items: [{ price }],
```
`plan`, and Plans generally, are deprecated (although backwards compatible).

Reply:
@dackers86 Could we adapt `items` to serialize the `line_items` data from the Firestore document? Their APIs are quite similar.